ocular/Cargo.toml

```
@@ -12,12 +12,23 @@ readme = "README.md"
 
 [dependencies]
 assay = "0.1"
+bip32 = "0.3.0"
```

The `bip32` crate has `bip39` support built-in:

```suggestion
bip32 = { version = "0.3.0", features = ["bip39"] }
```

---

Actually it's enabled by default, so this is unnecessary